### PR TITLE
Feat/GitHub created at

### PR DESCRIPTION
**Pull request by ravikumar1002**
### What is the change?
 Added `github_created_at` key: value for a new joiner in the user info

### Database changes:
 Yes but for new joiners, Updating  existing users I will create a new ticket for that after getting merge this PR https://github.com/Real-Dev-Squad/website-backend/pull/1044

### Test Result:
 
image
image

### Demo `*github_created_at` key: 
image

### CODE REVIEW

File: controllers/auth.js [hunk 1]
```diff
@@ -49,10 +49,10 @@ const githubAuthCallback = (req, res, next) => {
         logger.error(err);
         return res.boom.unauthorized("User cannot be authenticated");
       }
-
       userData = {
         github_id: user.username,
         github_display_name: user.displayName,
+        github_created_at: user._json.created_at,
```
Comment (sahsisunny) [inline controllers/auth.js:55]: Can we store the timestamp in Unix timestamp for consistency?

**Review by sahsisunny — CHANGES_REQUESTED** [review]
Where is the test?

File: controllers/auth.js (hunk 1 above)
Comment (ravikumar1002) [reply controllers/auth.js:55]: OK, I will change it

**Comment by ravikumar1002** [comment]
> Where is the test?

How to write test for this one line

File: controllers/auth.js (hunk 1 above)
Comment (ravikumar1002) [reply controllers/auth.js]: done

**Comment by sahsisunny** [comment]
> > Where is the test?
> 
> How to write test for this one line

Go to [google.com](https://google.com) and search "how to write test?"

**Review by sahsisunny — APPROVED** [review]
LGTM

File: test/fixtures/auth/githubUserInfo.js [hunk 2]
```diff
@@ -101,5 +101,49 @@ module.exports = () => {
...
+        twitter_username: "kumarravi1002",
+        public_repos: 47,
+        public_gists: 0,
+        followers: 9,
+        following: 13,
+        created_at: "2021-07-02T16:40:44Z",
+        updated_at: "2023-07-27T00:12:32Z",
+      },
+    },
   ];
```
Comment (vinit717) [inline test/fixtures/auth/githubUserInfo.js:148]: Here no need to create one more fixture 
add to previous data itself

File: test/fixtures/user/user.js [hunk 3]
```diff
@@ -343,5 +343,20 @@ module.exports = () => {
...
+      github_display_name: githubUserInfo[2].displayName,
+      github_created_at: Number(new Date(githubUserInfo[2]._json.created_at).getTime()),
+      roles: {
+        member: false,
+        in_discord: true,
+      },
+      incompleteUserDetails: false,
+      updated_at: Date.now(),
+      created_at: Date.now(),
+    },
```
Comment (vinit717) [inline test/fixtures/user/user.js:360]: here also

File: test/fixtures/auth/githubUserInfo.js (hunk 2 above)
Comment (ravikumar1002) [reply test/fixtures/auth/githubUserInfo.js:148]: ok

File: test/fixtures/user/user.js (hunk 3 above)
Comment (ravikumar1002) [reply test/fixtures/user/user.js:360]: ok, so I will remove this

**Review by vinit717 — APPROVED** [review]

**Pull request merged**